docs/releases/5.0.txt

```diff
@@ -588,6 +588,11 @@ Miscellaneous
 
 * The minimum supported version of ``docutils`` is increased to 0.19.
 
+* Due to a commit meant to address a SQLite crash and a DDOS vector on
+  Postgres, non-wrapped arithmetic against integer fields might require
+  adjustments, like explicit wrapping with ``ExpressionWrapper``, see
+  (:ticket:`35147`).
```

Comment:
We don't refer ticket number in a feature release notes. Maybe:
```suggestion
* Filtering querysets against overflowing integer values now immediately
  returns an empty queryset. As a consequence, you may need to use
  ``ExpressionWrapper`` to :ref:`explicitly wrap <using-f-with-annotations>`
  arithmetic against integer fields in such cases.
```

Reply:
Thanks for the comment Felix. I guess the referrence to a ticket are only for Bugfixes. I rewrite the note per your suggestion.